apiclient/harvester_api/managers.py

```diff
@@ -203,6 +203,13 @@ def delete(self, name, namespace=DEFAULT_NAMESPACE, *, raw=False):
         path = self.PATH_fmt.format(uid=f"/{name}", ns=namespace)
         return self._delete(path, raw=raw)
 
+    def export(self, volume_name, image_name, namespace=DEFAULT_NAMESPACE, *, raw=False):
+        export_spec = {"displayName": image_name,
+                       "namespace": namespace, "storageClassName": "harvester-longhorn"}
```

Comment:
for the method definition, consider the document would like:
```
class harvester_api.VolumeManger
    export(volume_name, image_name, namespace='default', *, raw=False)
        # the description of the method
```

I would prefer to use `name` instead of `volume_name` for the first argument.

`storageClassName` should be one of the argument, same as the UI provides

Reply:
Update related lines according to comment, add `storage_class` for `storageClassName`